app/views/shared/_test_banner.html.haml

```diff
@@ -0,0 +1,10 @@
+.banner
+  .banner__content
+    %p
+      Hello and welcome! We have been working hard to improve the NZSL dictionary website.
+      We would love your feedback. As you go through the website, you will see questions pop up.
+      At the end of your visit, please click on
+      %a{ href: "https://insights.hotjar.com/s?siteId=587430&surveyId=26313" } this link
+      to give us more feedback. Thank you!
+    %p
+      %a{ href: "https://youtu.be/D2d-_OVhdog" } View this message in NZSL
```

Comment:
also -- should it be a new window or some other warning that you're leaving this website..?